node/lib/util/write_repo_ast_util.js

```diff
@@ -668,11 +670,16 @@ git -c gc.reflogExpire=0 -c gc.reflogExpireUnreachable=0 \
             yield writeRepo(subRepo, openSubAST);
         }
     }
-
+    const reverseUrlMap = {};
+    Object.keys(urlMap).forEach(url => {
+        reverseUrlMap[urlMap[url]] = url;
+    });
     return {
         repos: resultRepos,
         commitMap: commitMaps.newToOld,
+        reverseMap: commitMaps.oldToNew,
```

Comment:
OK, but I pushed it in a separate commit because I was using that name in other places that had not been changed.